src/app/pages/network-health/network-health.css

```
@@ -344,3 +347,76 @@ app-slider-carousel {
     padding: 40px 2rem 0 2rem;
   }
 }
+
+.glossary{
+  display: flex;
+  flex-direction: row;
+  gap: 2px;
+}
+
+/* Dialog container */
+::ng-deep .mat-dialog-container {
```

Biome (2.1.2)

[error] Unexpected unknown pseudo-elements: ng-deep

See MDN web docs for more details.
Use a known pseudo-elements instead, such as:


(lint/correctness/noUnknownPseudoElement)